languages/cpp/exercises/concept/numbers/.meta/example.cpp

```diff
@@ -0,0 +1,33 @@
+#include "numbers.h"
+
+namespace assembly_line {
+
+constexpr uint32_t PRODUCTION_RATE_DEFAULT_SPEED = 221;
```

Comment:
`<cstdint>` defines `std::uint32_t`, `<stdint.h>` defines `::uint32_t`.
Please signed types for arithmetic operations, see [ES.100-102 of the C++ Core Guidelines](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es100-dont-mix-signed-and-unsigned-arithmetic).
[ES.9 of the C++ Core Guidelines](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es9-avoid-all_caps-names) says: "*Avoid ALL_CAPS names*".